php/meterpreter/ext_server_stdapi.php

```diff
@@ -1266,6 +1266,37 @@ function stdapi_registry_set_value($req, &$pkt) {
 }
 }
 
+if (!function_exists('stdapi_net_config_get_arp_table')) {
+if (is_linux()) {
+    register_command('stdapi_net_config_get_arp_table');
+}
+function stdapi_net_config_get_arp_table($req, &$pkt) {
+    if !(is_linux()) {
+        return ERROR_FAILURE;
+    }
```

Comment:
I was getting a syntax error with this.

```suggestion
    if (!is_linux()) {
        return ERROR_FAILURE;
    }
```